src/issue-filing/common/issue-filing-url-string-utils.ts

```diff
@@ -15,9 +15,7 @@ const getTitle = (data: CreateIssueDetailsTextData): string => {
         prefix = prefix + ': ';
     }
 
-    const selectorLastPart = getSelectorLastPart(data.ruleResult.selector);
-
-    return `${prefix}${data.ruleResult.help} (${selectorLastPart})`;
+    return `${prefix}${data.rule.description} (${data.element.conciseName})`;
 };
 
 const getSelectorLastPart = (selector: string): string => {
```

Comment:
In a different PR we may want to move `getSelectorLastPart` out of this file because the other functions are platform-agnostic whereas this is web-specific.